cwl_tes/tes.py

```diff
@@ -389,13 +405,31 @@ def run(self,
         )
         log.info(pformat(task))
 
+        if self.lib:
```

Comment:
Why not add this conditional check after the logging lines in the original code block rather than duplicate this whole code block?